layer1/transaction/backend.go

```diff
@@ -43,7 +43,7 @@ func (fs *FuncSelector) UnmarshalJSON(input []byte) error {
 	return hexutil.UnmarshalFixedJSON(FuncSelectorT, input, fs[:])
 }
 
-// Internal struct to keep track of transactions that are being monitoring
+// info is an internal struct to keep track of transactions that are being monitoring
 type info struct {
```

Comment:
(I know you're just modifying the comments here, but since we're here...)

This might make more sense being named `monitored`, which would allow the comment to be:
```golang
// A monitored transaction.
```

Which is a little more specific than just `info`.

Reply:
monitored seems like a good name!